dozer-core/src/storage/lmdb_storage.rs

```diff
@@ -17,7 +17,7 @@ const DEFAULT_MAX_READERS: u32 = 256;
 const DEFAULT_MAX_MAP_SZ: size_t = 1024 * 1024 * 1024;
 
 pub struct LmdbEnvironmentManager {
-    inner: Environment,
+    pub inner: Environment,
```

Comment:
This field is not `pub` on purpose, because once it's `pub`, users can use it to create transactions and use them without proper locking, which can cause data races. Users should always rely on `create_txn` to create new transactions.

Reply:
got it will remove the test then

Reply:
Is this design blocking you from implementing a test? Which one?

Reply:
Please do this `let mut tx = storage.create_txn().unwrap();`

Reply:
it's ok - just a simple unit test in the `aggregation_tests` but fixed it using new Environment attribute ;)